components/organisms/Education.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { TimelineEvent } from '../molecules/Timeline';
+import { BookOpenIcon } from '@heroicons/react/24/solid';
+
+
+interface EducationEntry {
+    name: string;
+    startDate: number;
+    endDate: number;
+    description?: string;
+    icon?: React.ReactNode;
+}
+
+const education: EducationEntry[] = [
+    {
+        name: 'Federal State Budget-Financed Educational Institution of Higher Education The Bonch-Bruevich',
+        startDate: 2022,
+        endDate: 2026,
+        description: 'Study at the university in the direction of "Technology".',
+        icon: <BookOpenIcon width={36} />,
+    },
+    {
+        name: 'Lyceum No. 5',
+        startDate: 2012,
+        endDate: 2022,
+        description: 'Secondary education with an emphasis in technical sciences.',
+        icon: <BookOpenIcon width={36} />,
+    }
+];
+
+export default function Education() {
+    const currentDate = new Date().getFullYear();
+
+    return (
+        <div className="md:container text-muted-foreground self-center pt-10 w-full relative">
```

Comment:
```suggestion
        <div className="md:container text-muted-foreground self-center pt-10 w-full relative">
```
Зачем тут text-muted-foreground md:container  w-full

Reply:
Ну, если честно, то это я тупо взял с Experience. А так, чтобы оттенок цвета текста был менее насыщенным.